StoreManagementService/src/PBJ.StoreManagementService.Business/Services/Abstract/ICommentService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using PBJ.StoreManagementService.Business.Dtos;
+
+namespace PBJ.StoreManagementService.Business.Services.Abstract
+{
+    public interface ICommentService
+    {
+        Task<List<CommentDto>> GetAmountAsync(int amount);
```

Comment:
It's possible to remove "Async" from method name because we often use async await in every method.